src/promptflow/promptflow/_sdk/_service/README.md

```diff
@@ -0,0 +1,39 @@
+# Prompt Flow Service
+This document will describle the usage of pfs(prompt flow service) CLI.
+
+### Install prompt flow service
+
+You can execute this command to install pfs as a service in your machine.
```

Comment:
Describe how this is auto starts.

Reply:
msi will install a per-user service, needn't to describe how to auto start.